javalib/src/main/scala/java/time/temporal/ChronoUnit.scala

```diff
@@ -24,41 +25,54 @@ final class ChronoUnit private (name: String, ordinal: Int, duration: Duration,
 }
 
 object ChronoUnit {
-  final val NANOS = new ChronoUnit("Nanos", 0, Duration.OneNano, false)
+  private final val isTimeBasedFlag = 1
+  private final val isDateBasedFlag = 2
+
+  final val NANOS =
+      new ChronoUnit("Nanos", 0, Duration.OneNano, isTimeBasedFlag)
```

Comment:
These should be indented by 2 spaces only (or new newline at all if they don't exceed 80)